src/tests/Microsoft.PowerFx.Interpreter.Tests/FileExpressionEvaluationTests.cs

```diff
@@ -41,7 +46,113 @@ public void InterpreterTestCase(ExpressionTestCase testCase)
                     Skip.If(true, prefix + msg);
                     break;
             }
-        } 
+        }        
+
+        [InterpreterTheory] //(Skip="NotReadyYet")]
+        [TxtFileData("ExpressionTestCases\\NotYetReady", "InterpreterExpressionTestCases", nameof(InterpreterRunner))]
+        public void InterpreterTestCase_NotReadyTests(ExpressionTestCase testCase)
+        {
+            // This is running against embedded resources, so if you're updating the .txt files,
+            // make sure they build is actually copying them over. 
+            Assert.True(testCase.FailMessage == null, testCase.FailMessage);
+
+            _runner = new InterpreterRunner();
+
+            var (result, msg) = _runner.RunAsync(testCase).Result;
+
+            var prefix = $"Test {Path.GetFileName(testCase.SourceFile)}:{testCase.SourceLine}: ";            
+
+            switch (result)
```

Comment:
>switch [](http://example.com/codeflow?start=12&length=6)

If you want to hack something to skip all tests containing blank, you could do something like:

if (testCase.Input.Contains("Blank()")) { result = TestResult.Skip; }

Reply:
I've a better idea on how to handle Blank() for all functions